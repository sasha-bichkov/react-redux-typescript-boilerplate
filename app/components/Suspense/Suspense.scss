@use "sass:color";

.SuspenseComponent .Loader {
  @include centerer;
}
