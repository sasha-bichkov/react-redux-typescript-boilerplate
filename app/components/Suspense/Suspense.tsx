import React, { FC } from 'react'

import Loader from '@Components/Loader'

import './Suspense.scss'

const Suspense: FC = () => {
  return (
    <div className="Suspense">
      <Loader />
    </div>
  )
}

export default Suspense
