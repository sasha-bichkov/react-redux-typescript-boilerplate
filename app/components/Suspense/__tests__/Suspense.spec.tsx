import React from 'react'
import { render } from '@testing-library/react'
import '@testing-library/jest-dom'

import Suspense from '../Suspense'

describe('Suspense', () => {
  it('matches snapshot', () => {
    const { asFragment } = render(<Suspense />)

    expect(asFragment()).toMatchSnapshot()
  })
})
