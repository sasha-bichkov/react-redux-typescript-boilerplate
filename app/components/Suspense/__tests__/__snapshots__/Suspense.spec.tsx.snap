// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Suspense matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="Suspense"
  >
    <div
      class="Loader"
    >
      <div />
      <div />
      <div />
      <div />
    </div>
  </div>
</DocumentFragment>
`;
